tests/src/kakarot/instructions/test_block_information.py

```diff
@@ -46,6 +46,6 @@ def test__exec_block_information(self, cairo_run, opcode, expected_result):
             }
         ),
     )
-    def test__exec_chain_id__should_return_mod_64(self, cairo_run):
+    def test__exec_chain_id__should_return_mod_53(self, cairo_run):
         output = cairo_run("test__exec_block_information", opcode=Opcodes.CHAINID)
-        assert output == hex(BIG_CHAIN_ID % 2**32)
+        assert output == hex(BIG_CHAIN_ID % 2**53)
```

Comment:
I think that this test belongs to kakarot unit test, for a `eth_chain_id` test.

Reply:
what do you mean? it should also be testing that the block info is correct imo

Reply:
block info tests that the opcode returns the chain id, but the opcode is not responsible for any kind of "special" computation around this value. So the test with `BIG_CHAIN_ID` doesn't belong to the opcode's unit tests